packages/orchestration/src/workflow/local-workflow.ts

```diff
@@ -215,45 +209,6 @@ export class LocalWorkflow {
     this.flow.appendAction(action, to, options)
     this.handlers.set(action, handler)
 
-    this.hasChanges = true
-    return this.flow
-  }
-
-  moveAction(actionToMove: string, targetAction: string): OrchestratorBuilder {
-    this.flow.moveAction(actionToMove, targetAction)
-
-    this.hasChanges = true
-    return this.flow
-  }
-
-  moveAndMergeNextAction(
-    actionToMove: string,
-    targetAction: string
-  ): OrchestratorBuilder {
-    this.flow.moveAndMergeNextAction(actionToMove, targetAction)
-
-    this.hasChanges = true
-    return this.flow
-  }
-
-  mergeActions(where: string, ...actions: string[]) {
-    this.flow.mergeActions(where, ...actions)
-
-    this.hasChanges = true
-    return this.flow
-  }
-
-  deleteAction(action: string, parentSteps?) {
-    this.flow.deleteAction(action, parentSteps)
-
-    this.hasChanges = true
-    return this.flow
-  }
-
-  pruneAction(action: string) {
-    this.flow.pruneAction(action)
-
-    this.hasChanges = true
```

Comment:
We're not extending the class OrchestratorBuilder here, we'll need these methods.

Reply:
Oups you are right, i removed more than i wanted, i ll change that 👌